osu.Game/Screens/Play/GameplayClockContainer.cs

```diff
@@ -205,22 +211,33 @@ bool IAdjustableClock.Seek(double position)
 
         void IAdjustableClock.Reset() => Reset();
 
-        public void ResetSpeedAdjustments()
-        {
-        }
+        public void ResetSpeedAdjustments() => throw new NotImplementedException();
```

Comment:
Why does this throw now rather than do nothing as before?

Reply:
I figure that it's better to be informed if this is called, considering it won't actually do what it is expected to.

Reply:
These types of changes scare me a bit, because [several](https://github.com/ppy/osu/blob/11854158a855c27d7283a57f6d4a9ff4754b1904/osu.Game.Tests/OnlinePlay/TestSceneCatchUpSyncManager.cs#L214-L216) [places](https://github.com/ppy/osu/blob/11854158a855c27d7283a57f6d4a9ff4754b1904/osu.Game/Screens/OnlinePlay/Multiplayer/Spectate/CatchUpSpectatorPlayerClock.cs#L51-L53) suppress this method to do nothing too. I guess we can go with it I'm not sure how to even begin checking that this won't crash in some cursed scenario.